src/js/tech/html5.js

```diff
@@ -195,7 +195,10 @@ class Html5 extends Tech {
     el.load = () => {
       const retval = oldLoad.call(el);
 
-      this.triggerSourceset(el.src || el.currentSrc);
+      // if `el.src` is set, that source will be loaded
+      // otherwise, we can't know for sure what source will be set, so,
```

Comment:
it could be good to mention that: `<source>` elements will be used if the `src` attribute isn't set, and we cannot track that because that happens async without a good way of interception.